Rakefile

```diff
@@ -0,0 +1,61 @@
+desc "Release a new project version"
+task :release do
+  require 'more_core_extensions/core_ext/hash/nested'
+  require 'pathname'
+  require 'yaml'
+
+  version = ENV["RELEASE_VERSION"]
+
+  if version.nil? || version.empty?
```

Comment:
If we're including more_core then we have activesupport, so might as well use `.blank?`